packages/components/src/base-control/style.scss

```diff
@@ -22,5 +22,5 @@
 }
 
 .components-base-control + .components-base-control {
-	margin-bottom: $grid-unit-20;
+	margin-top: $grid-unit-20;
```

Comment:
This caused the following regression:

<img width="273" alt="Screenshot 2020-03-11 at 10 30 55" src="https://user-images.githubusercontent.com/1204802/76402230-6be11480-6383-11ea-8d49-37131cc3efcb.png">

What effort did it solve to make the margin a top margin rather than a bottom margin? Because I'm looking at fixing this in a separate PR and would like to avoid re-introducing the same issue you solved in making this change.

Reply:
Well this rule is meant to style consecutive controls right to add space between them. and in general consecutive controls are one under the other (unless some flex positioning in the parent is involved) which means the margin should be top since the selector targets the second element.

This was causing weird margins between the multiple entities in the popover.

Reply:
> Well this rule is meant to style consecutive controls right to add space between them.

I don't think that was the intent of the rule, but if it was, then I agree the margin should be at the top. I'll look for a fix that doesn't style multiple consecutive base controls in this way.

Reply:
I can't think of any other reason why this rule could exist, and also I wonder if it's me that introduce this rule from the first release of BaseControl :P .

Reply:
In #20782 I removed it, an so far not seeing any issues. 

As far as I can tell, the rule was introduced here: https://github.com/WordPress/gutenberg/pull/13181/files#diff-7f4a5101433bca0f9e460c45196b772fR24

From the conversation, it's not immediately clear why that particular rule was necessary. But given additional sidebar polish that is meant to happen soon, I would expect these spacing rules to be revisited regardless.